shell/platform/embedder/embedder.h

```diff
@@ -170,6 +169,8 @@ typedef enum {
   ///
   /// Only applicable when kFlutterSemanticsFlagIsTextField flag is on.
   kFlutterSemanticsFlagIsReadOnly = 1 << 20,
+  /// Whether the semantics node represents a link.
+  kFlutterSemanticsFlagIsLink = 1 << 21,
```

Comment:
Should this be 22 like the rest? I noticed `isFocusable` is not represented here.
cc/ @gspencergoog

Reply:
Yes, I that was an oversight, thanks!  Created https://github.com/flutter/engine/pull/12700 to fix that.